cmd/construct.go

```diff
@@ -0,0 +1,129 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+	"github.com/manifoldco/promptui"
+	"github.com/spf13/cobra"
+)
+
+/* WARNING - THIS WILL WRITE THE README IN THE DIRECTORY RUNNED
+ * PLEASE HAVE CAUTION
+ */
+
+// Structure of the README
+
+type PromptStructure struct {
+	ErrorMsg string
```

Comment:
i don't see any usages of ErrorMsg anywhere, any specific reason it got added in the PromptStructure ?

Reply:
I can try remove it and see what happens but saw it in cobra documentation.

Reply:
Can't find this struct either, assuming it also got removed.